src/ui/prompts/variablePrompts.ts

```diff
@@ -1,18 +1,25 @@
 import { Variable } from '../../api/schemas'
 import { fetchVariables, variableTypes } from '../../api/variables'
 import { ListQuestion, Question } from 'inquirer'
+import chalk from 'chalk'
+import { PromptResult } from './types'
 
 type VariableChoice = {
     name: string,
     value: Variable
 }
 
+export type VariablePromptResult = {
+    variable: VariableChoice['value']
+} & PromptResult
+
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const variableChoices = async (input: Record<string, any>):Promise<VariableChoice[]> => {
     const variables = await fetchVariables(input.token, input.projectKey)
     const choices = variables.map((variable) => {
+        const name = variable.name ? `${variable.name} ${chalk.gray(`(${variable.key})`)}` : variable.key
```

Comment:
I believe we realized we need to use `dim` otherwise Jason can't see the logs 👀 
```suggestion
        const name = variable.name ? `${variable.name} ${chalk.dim(`(${variable.key})`)}` : variable.key
```

Reply:
good catch!